docs/pages/management/operations/self-hosted-automatic-agent-updates.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Setting up self-hosted automatic agent updates
+description: How to setup automatic agent update for self-hosted Teleport
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["enterprise"]}
+  scopeOnly={true}
+  min="13.0"
+>
+  Automatic agent update is available starting from Teleport `13.0`.
+</Details>
+
+Starting with Teleport 13, agents running in systemd or Kubernetes environments
+can be automatically updated. The [automatic updates architecture
+page](../../architecture/automatic-agent-update.mdx) describes how agent
+updating works.
+
+This guide covers how to set up the automatic update infrastructure. If this is
+already done, or you are a Teleport Cloud user, you can directly
+[enroll agents into automatic updates](./enroll-agent-into-automatic-updates.mdx).
+
+## Requirements
+
+- Self-hosted Teleport cluster running.
+- `tctl` admin rights.
+- Either:
+  - a public S3/GCS bucket,
+  - a web server accessible from all agents with valid TLS certificates.
+
+## Step 1/3. Create release channel files
+
+A release channel contains two pieces of information: the targeted version
+and if the update is critical. Updaters subscribe to a release channel and will
+update to the provided version if possible. If the update is critical, updaters
+will ignore the maintenance schedule and update as soon as possible.
+
+Create a directory for the new release channel `current`.
+
+```code
+$ mkdir current/
+```
+
+Make the `current` release channel target the version (=teleport.version=):
+
+```code
+$ echo -n "(=teleport.version=)" > current/version
+```
+
+And mark the update as not critical:
+
+```code
+$ echo -n "no" > current/critical
```

Comment:
Almost missed this! Without another `>`, won't this overwrite?

```
❯ echo -n "line 1" > example
❯ echo -n "line 2" > example
❯ cat example
line 2
```

```suggestion
$ echo -n "no" >> current/critical
```

And I'm not sure the best way to adjust to resolve, but using `-n` on both commands yields a single line, which I don't think is intended:

```
❯ echo -n "line 1" > example
❯ echo -n "line 2" >> example
❯ cat example
line 1line 2%
```

Reply:
Both commands are writing to different files (at least they're supposed to), the first one wites to `current/version` and the second one writes to `current/critical`. The file should have no newline at the end (updaters should be resilient but I'm more comfortable not testing the limits of the system). I think overwriting is the sane behaviour here as it allows the user to mistakenly run the command twice.